src/components/App.jsx

```diff
@@ -8,12 +8,14 @@ import SigninPage from 'pages/SigninPage/SigninPage';
 import MainPage from 'pages/MainPage/MainPage';
 import { selectLoggedIn } from 'redux/selectors/users.selectors';
 import { refreshUser } from 'redux/actions/users.actions';
+import { Header } from './Header/Header';
 
 const unathorisedRoute = (
   <Route path="/">
     <Route index element={<WelcomePage />}></Route>
     <Route path="/register" element={<RegisterPage />}></Route>
     <Route path="/signin" element={<SigninPage />}></Route>
+    <Route path="/header" element={<Header />}></Route>
```

Comment:
Header nie powinien mieć route, to tylko komponent, który ma się wyświetlać w szablonie

Reply:
tak zapomniałem go usunąć. zrobiłem go do testów